tests/test_response.py

```diff
@@ -81,3 +74,14 @@ def test_response_option_mimetype_init(monkeypatch):
     assert ro.static_media_types['.js'] == 'text/javascript'
     assert ro.static_media_types['.json'] == 'application/json'
     assert ro.static_media_types['.mjs'] == 'text/javascript'
+
+
+@pytest.mark.parametrize('content', [b'', b'dummy content'])
+def test_response_set_stream(resp, content):
+    stream = BytesIO(content)
+    content_length = len(content)
+
+    resp.set_stream(stream, content_length)
+
+    assert resp.stream == stream
```

Comment:
Thanks, looks great now bar one small nitpick: it is not very meaningful to compare instances of `BytesIO` this way. Maybe asserting `assert resp.stream is stream` would be more precise?

Reply:
Done 👍🏻